### ISSUE TITLE
``dataframe.repr()`` doesn`t work correctly at MODIN_STORAGE_FORMAT=omnisci

### ISSUE THREAD

**Issue by Rubtsowa** [modin-project/modin#4069]
When I run:
```
import pandas
import modin.pandas as pd
from modin.pandas.test.utils import (random_state, RAND_LOW, RAND_HIGH)
import modin.config as cfg
cfg.StorageFormat.put('omnisci')

frame_data = random_state.randint(RAND_LOW, RAND_HIGH, size=(1000, 100))
pandas_df = pandas.DataFrame(frame_data)
modin_df = pd.DataFrame(frame_data)
assert repr(pandas_df) == repr(modin_df)
```
I have error:
```
AssertionError: assert '     0   1  ... 100 columns]' == '    0   1   ... 100 columns]'
-     0   1   2   3   4   5   6   7   8   ...  91  92  93  94  95  96  97  98  99
+      0   1   2   3   4   5   6   7   8   ...  91  92  93  94  95  96  97  98  99
? +
- 0   93  99   5  57  77  56   2  23  18  ...  32  56  36  56  97  61   2  29  66
+ 0    93  99   5  57  77  56   2  23  18  ...  32  56  36  56  97  61   2  29  66
?  +
- 1   62  81  22  71  72  65  60   1  26  ...  46  85  51  40  91  46  72  86  70...

...Full output truncated (37 lines hidden), use '-vv' to show
```

**Comment by vnlitvinov** [comment]
@modin-project/modin-omnisci  could someone please check if this still happens?

**Comment by AndreyPavlenko** [comment]
This issue has been fixed by #4834
